webapp/src/widgets/icons/brokenFile.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react'
+
+import CompassIcon from './compassIcon'
+
+export default function BrokenFile(): JSX.Element {
+    return (
+        <CompassIcon
+            icon='file-image-broken-outline'
+            className='BrokenFile'
+        />
+    )
+}
```

Comment:
Is this still necessary? I thought we could just use <CompassIcon> directly.

Reply:
Sorry, didn't get it. You mean directly use `<CompassIcon/>` instead of creating this file?

Reply:
Yeah. Asking to make sure I'm doing it correctly.

Reply:
Here's the card for reference - https://community-daily.mattermost.com/boards/workspace/b1ni7nprjpfwpb566k1gk8z9nw/886d2280-e88e-4065-8c85-1b62578f192e/f30dcb19-a0e8-45d2-8768-a9489f6e7fe8/ch47b5ot8bfrq5jpktjhccu3rqw